alerting/src/main/resources/org/opensearch/alerting/alerts/alert_mapping.json

```diff
@@ -74,6 +74,9 @@
         }
       }
     },
+    "workflow_execution_id": {
```

Comment:
Do we want this to be generic and be `execution_id`? It will be harder to change it to `execution_id` later on and might be best to do that if we want to later correlate an alert to an execution for normal monitors.

Reply:
Agreed that there is value in keep execution_id generic to extend the capability for monitors executing outside workflows

i think the best solution is - for **alerts** and **findings** we should have three fields
- `monitor_id` (empty for chained alerts)
- `workflow_id` (empty for monitors executing independently outside workflows)
- `execution_id` (empty today for monitors executing independently outside workflows but can be populated tomorrow)

This would make it easy to query for alerts page in dashboard where as a user i want to see ONLY chained_alerts and independent monitor alerts (`get all alerts where monitor_id is "" or workflow_id is null`)

And in chained alert detail page for a chained alert I would want to see the delegate monitor alerts - `query by execution id and workflow_id`